CMake/Dependencies/libwebsockets-CMakeLists.txt

```diff
@@ -51,7 +51,7 @@ ExternalProject_Add(project_libwebsockets
         -DLWS_WITH_STATIC=1
         -DLWS_WITH_SHARED=${LWS_WITH_SHARED}
         -DLWS_WITH_MBEDTLS=${LWS_WITH_MBEDTLS}
-        -DLWS_WITH_MINIMAL_EXAMPLES=1
+        -DLWS_WITH_MINIMAL_EXAMPLES=0
```

Comment:
Why are we changing this?

Reply:
We dont need to build minimal examples for LWS build. It just adds on to the build time.